packages/terra-spacer/docs/README.md

```diff
@@ -0,0 +1,71 @@
+# Terra Spacer
+
+This component is used to provide margin and/or padding space between two components based on the given values.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-spacer`
+  - `yarn add install terra-spacer`
+
+## Naming Convention
+
+| String Name   | Computed PX Value |
+| ------------- | -------------     |
+| none          | 0px               |
+| small-2       | 3px               |
+| small-1       | 5px               |
+| small         | 7px               |
+| medium        | 10px              |
+| large         | 12px              |
+| large+1       | 15px              |
+| large+2       | 20px              |
+| large+3       | 30px              |
+| large+4       | 50px              |
+
+## Usage
+
+```jsx
+import React from 'react';
+import Spacer from 'terra-spacer';
+
+// Small view ports
+
+<Spacer marginTop="small" marginLeft="small-1" paddingTop="small-2">
+  <Image src="someImageSource" alt="example image" />
+<Spacer/>
+
+// Medium view ports
+
+<Spacer marginLeft="medium" marginRight="medium">
+  <Image src="someImageSource" alt="example image" />
+<Spacer/>
+
+// Large and very large view ports
+
+<Spacer paddingTop="large" paddingBottom="large+1" paddingLeft="large+2" marginLeft="large+4" marginRight="large+3">
+  <Image src="someImageSource" alt="example image" />
+<Spacer/>
+
+// Responsive spacing
+
+// sets magin-top to 50px for view port sizes >= 768px, for view port sizes < 768px sets margin-top to 5px
+
+<Spacer marginTop="small-1" bps={{ 768: { marginTop: 'large+4' } }} >
+  <Image src="someImageSource" alt="example image" />
+</Spacer>
+```
+
+## Note
+
+This component uses ** react-bps package mobile first strategy ** for ** responsiveness **. Find more information on how it is used [here](https://www.npmjs.com/package/react-bps).
```

Comment:
Your formatting here isn't correct. You have to remove the spaces between the asterisks and the words that you want bold. 
**react-bps package mobile first strategy** for **responsiveness**.